### DISCUSSION TITLE
Shortcut to populate location list with results

### DISCUSSION THREAD

**Discussion by SuperCuber** [nvim-telescope/telescope.nvim#1375] [Q&A] [answered]
Say I use `:Telescope live_grep`, and I typed some stuff and now I'm happy with the results previewed to me and want to go through each one of them. It would be really nice if I could press some shortcut and telescope would populate my location/quickfix list with the results so I could go through all of them or `:cdo` etc. Is there a way to do this?

**Comment by Conni2461** [comment, marked as answer]
`<C-q>` to pipe all results to qflist. `<M-q>` to pipe all selected results to qflist.

We also have a smart version that isnt mapped on default tho. (pipe all if no selected, otherwise pipe selected)

```lua
telescope.setup {
  defaults = {
    mappings = {
      i = {
        ["<C-q>"] = actions.smart_send_to_qflist + actions.open_qflist,
      }
    }
  }
}
```
For more information on mappings see `:help telescope.defaults.mappings`

Also for loclist see. `:help actions.send_to_loclist`

**Reply by SuperCuber** [reply]
Very cool, happy to see it was builtin already! I'm looking at `:help telescope.defaults.mappings` but I don't see a list of the actual default mappings... Is there such a list documented somewhere?

**Reply by Conni2461** [reply]
Just an informal one: https://github.com/nvim-telescope/telescope.nvim#default-mappings

Otherwise: https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/mappings.lua#L11-L63
